deployment_config/prod_vars.yml

```diff
@@ -1,12 +1,9 @@
 AUTH_BASE: https://uat.hsesinfo.org
-AUTH_CLIENT_ID: ((AUTH_CLIENT_ID))
-AUTH_CLIENT_SECRET: ((AUTH_CLIENT_SECRET))
```

Comment:
I removed any vars from these files that are set via CircleCI configs. Those can go straight from the config.yml command line to manifest.yml